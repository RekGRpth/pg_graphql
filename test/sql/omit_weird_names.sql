begin;

    savepoint a;

    create table "@xyz"( id int primary key);
    select name from graphql.type where entity = '"@xyz"'::regclass;

    rollback to savepoint a;

    create table xyz( "! q" int primary key);
    select name from graphql.field where entity = 'xyz'::regclass and meta_kind = 'Column';

    rollback to savepoint a;

rollback;
