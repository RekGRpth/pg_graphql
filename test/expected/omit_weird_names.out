begin;
    savepoint a;
    create table "@xyz"( id int primary key);
    select name from graphql.type where entity = '"@xyz"'::regclass;
 name 
------
(0 rows)

    rollback to savepoint a;
    create table xyz( "! q" int primary key);
    select name from graphql.field where entity = 'xyz'::regclass and meta_kind = 'Column';
 name 
------
(0 rows)

    rollback to savepoint a;
rollback;
